sdr/concepts_and_interactions.md

```diff
@@ -0,0 +1,202 @@
+# Concepts, inter-service interactions, gems
+
+_A bit more focused on how the services fit together and why._
+
+## types of Digital Objects
+_We started with more, but we're kinda down to three_
+
+(ask Andrew)
+
+* APO - _administrative (or admin) policy object_
+* Collection - _a collection of items, belonging to an APO_
```

Comment:
The APO is irrelevant here. All objects (items, collections an APOs) are required to have one APO.